sportsreference/mlb/roster.py

```diff
@@ -1438,12 +1438,13 @@ class Roster:
     def __init__(self, team, year=None, slim=False):
         self._team = team
         self._slim = slim
+        self.coach = None
```

Comment:
To be consistent, and to add proper documentation where appropriate, we'd want to keep this private to this module's scope, so this should be `self._coach = None`.

Also, there should be a method with a `@property` decorator in this module, similar to the others below, which describes the variable. This automatically adds the property to the documentation, and exposes it to all users of the API. This would look similar to the following:

```python
@property
def coach(self):
    """
    Returns a ``string`` of the coach's name during the season, such as 'AJ Hinch' for the Houston Astros.
    """
    return self._coach
```